spinta/datasets/backends/dataframe/commands/read.py

```diff
@@ -362,14 +388,18 @@ def getall(
                 "pkeys": _get_pkeys_if_ref(prop)
             }
 
-    for params in iterparams(context, model):
-        meta = _get_dask_dataframe_meta(model)
-        df = dask.bag.from_sequence([base]).map(
-            _get_data_json,
-            source=model.external.name,
-            model_props=props
-        ).flatten().to_dataframe(meta=meta)
-        yield from _dask_get_all(context, query, df, backend, model, builder)
+    model_formatted_bases = []
+    for formatted_base in bases:
+        for params in iterparams(context, model.manifest, model.params):
+            model_formatted_bases.append(formatted_base.format(params))
```

Comment:
bases are now generators, but it only works for `xml` and `json`, since it has custom file read, while `csv` uses `dask` `read_csv` which only support `path` or `list`, so for `csv`, bases is converted to list.